### ISSUE TITLE
Make notebooks demonstrating effects of choices

### ISSUE THREAD

**Issue by NiallJeffrey** [NiallJeffrey/DeepMass#2]
- learning rate
- batch normalisation
